stl/inc/vector

```diff
@@ -477,17 +618,20 @@ public:
 
     _CONSTEXPR20 explicit vector(_CRT_GUARDOVERFLOW const size_type _Count, const _Alloc& _Al = _Alloc())
         : _Mypair(_One_then_variadic_args_t{}, _Al) {
+        _Asan_annotater<_Has_minimum_allocation_alignment<vector>> _Annotater(*this);

```

Comment:
Can we please go for a shorter typedef that we use once. That is quite a mouth full

Reply:
Hm, the problem I run into then is handling incomplete types. If I don't write out the whole type explicitly in the member functions I get errors like
```In file included from C:\src\STL\tests\std\tests\Dev11_0000000_user_defined_literals\test.cpp:11:
In file included from C:\src\STL\amd64\out\inc\chrono:30:
C:\src\STL\amd64\out\inc\vector(463,33): error: invalid application of 'alignof' to an incomplete type 'typename vector<Elem>::value_type' (aka 'Elem')
    disjunction_v<bool_constant<alignof(typename _Vec::value_type) >= 8>,```

Reply:
I could make it a macro.